### ISSUE TITLE
Section Discussion 

### ISSUE THREAD

**Issue by simnh** [simnh/addressing_challenges_oemof#50]

**Comment by cswh** [comment]
what are the details of this task?

**Comment by ckaldemeyer** [comment]
I think this is closely related to https://github.com/simnh/addressing_challenges_oemof/issues/34.

* Section "Discussion" - WHAT COULD POTENTIALLY BE ADDRESSED THROUGH THE FRAMEWORK? / WHAT COULD BE ADDRESSED THROUGH OTHER APPROACHES AND DEVELOPMENTS? / WHAT COULD PROBABLY NEVER BE ADDRESSED?
  * Why can certain challenges not be tackled?
  * What are the difficulties in addressing this challenge?
  * Are the difficulties problem-specific or could they be addressed by potential future developments within oemof?
  * What has been contributed by others?

**Comment by ckaldemeyer** [comment]
I just think that the sections "Addressed challenges" and "Discussion" should be thought in one big bracked.

The first holds the challenges that can be addressed right now and the second discusses everything beyond.

We can then still move the paragraphs and sections..

**Comment by fraukewiese** [comment]
For the discussion, I like the suggestions by Cord, I extended these thoughts. As before in the "Addressed challenges" section, it can be first discussed along the three characteristics: open, collaborative, structural, as 
(1) open lays the foundations for all five mentioned challenges
(1.1) and many models are going that way
(1.2) but there are still barriers
(2) collaborative software dev. contributes further to the five mentioned challenges
(2.1) and more and more tools that make it possible (e.g. github)
(2.2) include stuff from your experience, what you recongnized you learn during this collab. work, github
(2.3) but still barriers (project oriented research, financing structure)... 
(3) structural ... is foundation for challenges in context with modelling techniques
(3.1) elaborate on that the structural stuff lays the foundation for challenges in context with modelling techiniques
(3.2) framework is good. but: some (what?) has to be solved on the application-level.
(4) There are challenges left, not possible to solve neither on framework nor on application level. On research organization level maybe, societal level...
(5) What future research should do
(6) come back from the big picture to oemof: what are the next things you want to do

**Comment by ckaldemeyer** [comment]
> For the discussion, I like the suggestions by Cord, I extended these thoughts. As before in the "Addressed challenges" section, it can be first discussed along the three characteristics: open, collaborative, structural,..

I guess we have different tastes here and should decide on this ;-)

**Comment by ckaldemeyer** [comment]
But it reflects the general structure of my idea! It's just the question if you go property or challenge-wise..

**Comment by simnh** [comment]
I think we should start with wirting down the contribution ( #34 ) and then see how the structure of the disucussion will look like. @ckaldemeyer will do a proposal for the contribution and I suggest we will follow this one.

**Comment by fraukewiese** [comment]
yep, I agree.

**Comment by fraukewiese** [comment]
- I tried to order the existing text according to the challenges and Cords suggested structure and added some points. But I had some difficulties to think the challenge by challenge structure in the discussion. To make it easier to shift around blocks, I wrote a % comment on the beginning of each paragraph which thought is in there.
- I did not dare to delete thoughts that were already there, so feel free to shorten and straighten
- Some text I have put in \textit{} when I did not know how it fits or for what I think rather fits to addressed challenges section #34 .
- I did not add references so there are just the ones that already were there. But I think it could be ok for the discussion section to have very few references.
- I am not really satisfied with my work, so see what you think is useful, I am not offended when you delete or change my thoughts :)
